.github/dependabot.yml

```diff
@@ -4,4 +4,9 @@ updates:
   directory: "/"
   schedule:
     interval: weekly
-  open-pull-requests-limit: 10
\ No newline at end of file
+  open-pull-requests-limit: 10
+- package-ecosystem: cargo
+  directory: "/"
+  schedule: 
+    interval: weekly
+  open-pull-requests-limit: 10
```

Comment:
I think we should also add [this](https://github.com/rust-vmm/vhost-device/blob/main/.github/dependabot.yml#L8). More info here: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#allow.

Reply:
Can you also leave a link for this part: `it will only open pull requests for semver incompatible updates`. if that's the case I was wondering if it will ever open PRs for indirect deps. What I would like to happen is that dependabot will open PRs (or at least announce us) if an indirect dep is vulnerable even if it is a compatible update.
This might be of interest in this type of scenario: https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates and https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates#managing-dependabot-security-updates-for-your-repositories.

Reply:
Regarding indirect dependencies: Since vm-allocator is a library crate, I don't think that applies. There is no Cargo.lock, so indirect dependencies are not tracked anywhere. Its a good thing to keep in mind for binary crates though, so thanks for pointing that option out to me!
For the second part, that was mostly me testing in my fork (e.g. adding new entries to Cargo.toml and seeing what dependabot ends up creating PRs for. It did not open any for outdated patch versions, or for outdated minor versions if the major is >= 1)